cloudformation/media-atoms-dynamo-code.json

```diff
@@ -15,8 +15,12 @@
             "Description": "Code audit dynamo table",
             "Type": "String",
             "Default": "audit-media-atom-maker-CODE"
+        },
+        "UploadTrackingAtomMakerCodeTable": {
```

Comment:
It looks like merging them  into the main one and keeping the same dynamo tables isn't possible just yet. I'm fairly sure it will create new tables if we merge them into the main one. We should double check but I vaguely remember this conversation with @philwills a couple weeks ago. 
So it's up to you - we could put all the dynamo cloudformation in this one file and it be the one thing outside of the main one?